packages/evo-marko/src/tags/evo-progress-spinner/examples/large.marko

```diff
@@ -0,0 +1 @@
+<evo-progress-spinner size="large" aria-label="Busy"/>
```

Comment:
Should we add `...input` here so it's adjustable in Storybook?

In general, should we be adding `...input` to these examples? I fear that devs might copy-paste them when spreading input is not actually a good idea, but maybe that's not justified

Reply:
Well we do that so it works in storybook....